konrad/surface.py

```diff
@@ -140,7 +140,7 @@ def adjust(self, *args, **kwargs):
 #TODO: Rename `SlabOcean`?
 class SurfaceHeatCapacity(Surface):
     """Surface model with adjustable temperature."""
-    def __init__(self, *args, depth=50., **kwargs):
```

Comment:
The same as for the changes in `konrad.cloud`. Adding a trailing white space was probably not intended anyhow, so if you want, you can reverse the change to make the PR more concise 😉